sdk/samples/iot/README.md

```diff
@@ -1,6 +1,9 @@
 # Azure Embedded C SDK IoT Samples
 
+## Table of Contents
+
 - [Azure Embedded C SDK IoT Samples](#azure-embedded-c-sdk-iot-samples)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I think I saw some PR (maybe from @danewalton) where the ToC was removed somewhere else, but I can't recall or find it anymore. Dane, do you remember? Looks like we do have ToC item in other places.

**Question:** Do we want a ToC entry in the ToC? We are already in this section. Why would I click it?

It's fine either way, just curious.

Reply:
yes because the TOC is autogenerated by the extension.